conformance_pack/s3.sp

```diff
@@ -234,12 +234,9 @@ control "s3_public_access_block_bucket_account" {
   query       = query.s3_public_access_block_bucket_account
 
   tags = merge(local.conformance_pack_s3_common_tags, {
-    fedramp_low_rev_4      = "true"
     fedramp_moderate_rev_4 = "true"
     ffiec                  = "true"
-    nist_800_171_rev_2     = "true"
     nist_800_53_rev_4      = "true"
-    nist_csf               = "true"
```

Comment:
Why were all the tags removed?

Reply:
We are removing this as tagging was incorrect .